### DISCUSSION TITLE
Can I rely on histogram:info: support to persist?

### DISCUSSION THREAD

**Discussion by oleglk** [ImageMagick/ImageMagick#7721] [Help] [answered]
### ImageMagick version

7.3

### Operating system, version and so on

Windows

### Description

A question:
In an old discussion thread shown below Mr. Antony claimed that histogram:info: feature may become obsolete. May it indeed happen?

====================
[Re: Display histogram](https://legacy.imagemagick.org/discourse-server/viewtopic.php?p=73815#p73815)
Post by [anthony](https://legacy.imagemagick.org/discourse-server/memberlist.php?mode=viewprofile&u=1663) » 2011-06-24T20:20:40-07:00
...
In the future we will probably remove histogram: output in favour of in-memory histogram processing. But if done it will be a IM version 7 change (in pre-alpha development)
====================

Thank you in advance,
Oleg.

**Comment by dlemstra** [comment, marked as answer]
That comment was made over 13 years ago and we still have the `histogram:` delegate. This didn't get removed in ImageMagick 7 and there are now no plans to remove this.
